simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -66,20 +66,20 @@ public PushGateway(String address) {
   }
 
   /**
-   * Pushes all metrics in a registry, replacing all those with the same job as the grouping key.
+   * Pushes all metrics in a registry, replacing all those with the same job and no grouping key.
```

Comment:
I know that.

There are eight methods:
```
1. push(CollectorRegistry registry, String job)
2. push(Collector collector, String job)

3. push(CollectorRegistry registry, String job, Map<String, String> groupingKey)
4. push(Collector collector, String job, Map<String, String> groupingKey)

5. pushAdd(CollectorRegistry registry, String job)
6. pushAdd(Collector collector, String job)

7. pushAdd(CollectorRegistry registry, String job, Map<String, String> groupingKey)
8. pushAdd(Collector collector, String job, Map<String, String> groupingKey)
```

Methods 1,2,5,6 should have similar documentation. I made the documentation for method 1 similar to what was there for methods 2,5,6.